swagger.yaml

```diff
@@ -1355,6 +1355,26 @@ components:
         status:
           $ref: '#/components/schemas/TaskStatus'
           description: Task status. TaskConnections is taken into account when updating the status
+        assigneeIds:
+          type: array
+          items:
+            type: string
+          description: List of assignee user IDs
+        userRole:
+          $ref: '#/components/schemas/UserRole'
+          description: User role suggested for the task
+        estimatedDuration:
+          type: string
+          description: Estimated duration of the task
+        estimatedReadiness:
+          type: string
+          description: Estimated readiness of the task
+        attachmentUrls:
+          type: array
+          items:
+            type: string
+            format: url
+          description: URLs of attachments related to the task
```

Comment:
I am thinking wether this is enough or should we create a separate attachment object? I incline to keeping it like that for simplicity sake... @kaeline what do you think?